example.js

```diff
@@ -55,9 +55,18 @@ aqara.on('gateway', (gateway) => {
         })
         break
       case 'sensor':
-        console.log(`  Sensor (temperature:${device.getTemperature()}C rh:${device.getHumidity()}%)`)
+        console.log(`  Sensor (temperature:${device.getTemperature()}C rh:${device.getHumidity()}%${device.getPressure() != null ? ' pressure:' + device.getPressure() + 'kPa': ''})`)
         device.on('update', () => {
-          console.log(`${device.getSid()} temperature: ${device.getTemperature()}C rh:${device.getHumidity()}%`)
+          console.log(`${device.getSid()} temperature: ${device.getTemperature()}C rh:${device.getHumidity()}%${device.getPressure() != null ? ' pressure:' + device.getPressure() + 'kPa' : ''}`)
+        })
+        break
+      case 'leak':
+        console.log(`  Leak sensor`)
```

Comment:
Can you do like the other sensors? The device should have a `isLeaking()` method and emit events *on update*, not initially.

Reply:
For some reason I thought the leak sensor is more similar to the switch but now your idea sounds more reasonable :)
Made the changes